File: src/main/kotlin/com/itmo/microservices/demo/order/impl/dao/CatalogItemRepository.java
package com.itmo.microservices.demo.order.impl.dao;

import com.itmo.microservices.demo.order.impl.entity.CatalogItem;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface CatalogItemRepository extends JpaRepository<CatalogItem, UUID> {
}

File: src/main/kotlin/com/itmo/microservices/demo/shoppingCartService/impl/repository/CatalogItemRepository.kt
package com.itmo.microservices.demo.shoppingCartService.impl.repository

import com.itmo.microservices.demo.shoppingCartService.impl.entity.CatalogItem
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface CatalogItemRepository : JpaRepository<CatalogItem, UUID> {
}